ExtraCredit/ExtraCredit.sln

```diff
@@ -0,0 +1,101 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.3.32929.385
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "ExtraCredit", "ExtraCredit\ExtraCredit.vcxproj", "{1627850E-B12C-4CD9-933D-80B4728E493C}"
+EndProject
```

Comment:
```suggestion
Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "ExtraCredit", "ExtraCredit\ExtraCredit.vcxproj", "{1627850E-B12C-4CD9-933D-80B4728E493C}"
	ProjectSection(ProjectDependencies) = postProject
		{0CA908AA-6B57-4992-9FAB-8C2AFD62A0B6} = {0CA908AA-6B57-4992-9FAB-8C2AFD62A0B6}
		{1DCCC73F-2CDD-4111-8C90-C05B28F001D9} = {1DCCC73F-2CDD-4111-8C90-C05B28F001D9}
	EndProjectSection
EndProject
```
I'd add dependencies on the 2 DerivedShapes DLLs, since you would need them at runtime.